lib/elixir/lib/task.ex

```diff
@@ -12,46 +12,46 @@ defmodule Task do
       res  = do_some_other_work()
       res + Task.await(task)
 
-  Tasks spawned with `async` can be waited on by their caller
+  Tasks spawned with `async` can be awaited on by their caller
```

Comment:
There was a commit that changed the `awaited` to `waited` here, but I changed it back so that the same word is everywhere in the module.